### DISCUSSION TITLE
How to add a record to PowerDNS Recursor's Cache?

### DISCUSSION THREAD

**Discussion by ackema** [PowerDNS/pdns#14685] [General]
API /api/v1/servers/localhost/cache/flush can erase record from recursor's cache.
How Can I add a record(A or AAA) to recursor's cache?

**Comment by omoerbeek** [comment]
1. Query the record, it will be cached.
3. Use `zoneToCache` to load a full zone into the cache: https://docs.powerdns.com/recursor/lua-config/ztc.html#zone-to-cache
